packages/data-portal-explore/src/components/FileTable.tsx

```diff
@@ -238,13 +238,21 @@ const gen3ManifestInstructions = (gen3manifestFile: string | undefined) => {
                     <ol>
                         <li>
                             Install the{' '}
-                            <a href="https://gen3.org/resources/user/gen3-client/#1-installation-instructions" target="_blank" rel="noopener noreferrer">
+                            <a
+                                href="https://gen3.org/resources/user/gen3-client/#1-installation-instructions"
+                                target="_blank"
+                                rel="noopener noreferrer"
+                            >
```

Comment:
this is not related to the fix. might be related to a previously failed prettier attempt.